README.md

```diff
@@ -195,36 +195,42 @@ Validates that a field has the same value as another.
 
 ## Writing your own validators
 
-Adding your own validator is super simple – there are no Base classes to extend! **Validators are just functions**. All you need to do is to create a function with the correct signature.
+To create your own validator, generate a new validator function using the validator blueprint, and add your own logic.
```

Comment:
Out of all the changes in the readme, this is the only one I'm on board with. The generator is just a convenience, the current explanation already does a good job of explaining what a validator is. The `is-positive` example isn't necessary and will likely be more distracting than anything.